samples/sensor/bmm150/src/main.c

```diff
@@ -10,10 +10,20 @@
 #include <zephyr/drivers/sensor.h>
 #include <stdio.h>
 
-void do_main(const struct device *dev)
+void main(void)
 {
-	int ret;
+	const struct device *dev = DEVICE_DT_GET_ANY(bosch_bmm150);
```

Comment:
nit: this is inconsistent with commit title, any reason why `_ONE` is not used? If `_ONE` is causing compile errors, there's likely an issue with `sample.yaml` filtering.